From c8d57a162c1d5962bf013a93e9841d45e9dacff1 Mon Sep 17 00:00:00 2001
From: Chen-Yu Tsai <wens@kernel.org>
Date: Wed, 25 Mar 2026 00:43:51 +0800
Subject: [PATCH 402/419] soc: sunxi: sram: Const-ify sunxi_sram_func data and
 references

sunxi_sram_func contains value mapping that do not change at runtime.

Const-ify them.

Signed-off-by: Chen-Yu Tsai <wens@kernel.org>
---
 drivers/soc/sunxi/sunxi_sram.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/soc/sunxi/sunxi_sram.c b/drivers/soc/sunxi/sunxi_sram.c
index 2ccaeacf8c70..2b75ecc9f712 100644
--- a/drivers/soc/sunxi/sunxi_sram.c
+++ b/drivers/soc/sunxi/sunxi_sram.c
@@ -181,7 +181,6 @@ static const struct sunxi_sram_data *sunxi_sram_get_match(struct device_node *np
 	const struct of_device_id *match;
 	const struct sunxi_sram_data *data;
 	const struct sunxi_sram_func *func;
-
 	if (!of_device_is_available(np))
 		return ERR_PTR(-ENODEV);
 
-- 
2.43.0

